Library/Homebrew/cask/dsl/conflicts_with.rb

```diff
@@ -25,6 +25,10 @@ def initialize(pairs = {})
           instance_variable_set("@#{key}", instance_variable_get("@#{key}").merge([*value]))
         end
       end
+
+      def to_a
```

Comment:
I think this should rather be a hash.

Reply:
conflicts_with is initialized with a hash that becomes @pairs in the class context. Should it just return the @pairs instance variable?

Reply:
It should merge the pairs into a hash, i.e.

```json
{ "cask": ["firefox-beta", "firefox-esr"] }
```

Reply:
The test failures don't seem to be related to this.